src/Microsoft.Health.Operations.Functions/DurableTask/IDurableContextExtensions.cs

```diff
@@ -75,6 +76,29 @@ public static async Task<DateTime> GetCreatedTimeAsync(this IDurableOrchestratio
         return status.CreatedTime;
     }
 
+    /// <summary>
+    /// Asynchronously gets the date and time that the orchestration instance was created in UTC.
+    /// </summary>
+    /// <param name="context">An orchestration context.</param>
+    /// <param name="retryOptions">Optional settings for allowing multiple attempts.</param>
+    /// <returns>
+    /// A task that represents the asynchronous retrieval operation. The value of the <see cref="Task{TResult}.Result"/>
+    /// property represents the date and time that the orchestration was created in UTC.
+    /// </returns>
+    public static async Task<DateTimeOffset> GetCreatedAtTimeAsync(this IDurableOrchestrationContext context, RetryOptions? retryOptions = null)
```

Comment:
DateTimeOffset is used in the DTFx isolated worker APIs instead of DateTime. Furthermore, the property name has changed. In this activity, I am attempting to shim the APIs in preparation.

Reply:
You'll see a number of additions of this DateTimeOffset "CreatedAt" property